app/views/supervisor/dashboard/index.html.slim

```diff
@@ -2,15 +2,28 @@ h1 = "Hi #{current_user.name}!"
 h2 This is your brand-new Supervisor Dashboard - with love from Team Cheesy
 p alpha version
 
-hr
-
 div.clearfix
   - supervised_teams.each do |t|
-      div.dashboard-box
-        h3 = link_to t.name, t
+    div.dashboard-box
+      section.team-title
+        h3 = link_to "Team " + t.name, t
+      section.dashboard
+        h4 Checks
         - if t.last_checked_at?
-          p = "Your latest check was #{ format_date(t.last_checked_at, :short) }"
+          p = "Your latest check was #{ format_date(t.last_checked_at, :short) }."
+        - else
+          p You didn't check upon them yet.
+        = simple_form_for Comment.new(team_id: t.id), url: supervisor_comments_path do |f|
+          == render 'supervisor/comments/form', f: f
+        - if t.comments.any?
+          p = render t.comments.recent
+        - else
+          p No comments.
```

Comment:
Can we delete this else clause? For the user, there is no difference between 'checks' and 'comments', they are both 'checks'. If there are no comments, it means they haven't checked on them, and the previous message ("You didn't check...") wil show.